cmd/operator/kodata/knative-serving/0.15.1/4-net-kourier.yaml

```diff
@@ -0,0 +1,340 @@
+apiVersion: v1
```

Comment:
Where is the good place to save all the network ingresses?
Shall we separate them from the knative-serving?

Reply:
Just saw the comments from Jim regarding this aspect.

Reply:
For one version of operator, do we only associate with one version of network ingress? :-(

Reply:
Good questions and I don't have a good answer to them necessarily. We've discussed in the meetings that we go with a filter-based approach but the comments feel like we actually want a version per ingress? I guess that'd change everything.